AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py

```diff
@@ -0,0 +1,159 @@
+"""
+
+class Tests:
+    creation_undo = (
+        "UNDO Entity creation success",
+        "UNDO Entity creation failed")
+    creation_redo = (
+        "REDO Entity creation success",
+        "REDO Entity creation failed")
+    hdri_skybox_entity_creation = (
+        "HDRi Skybox successfully created",
+        "HDRi Skybox failed to be created")
+    hdri_skybox_entity_component_added = (
+        "Entity has a HDRi Skybox component",
+        "Entity failed to find HDRi Skybox component")
+    enter_game_mode = (
+        "Entered game mode",
+        "Failed to enter game mode")
+    exit_game_mode = (
+        "Exited game mode",
+        "Couldn't exit game mode")
+    is_visible = (
+        "Entity is visible",
+        "Entity was not visible")
+    is_hidden = (
+        "Entity is hidden",
+        "Entity was not hidden")
+    entity_deleted = (
+        "Entity deleted",
+        "Entity was not deleted")
+    deletion_undo = (
+        "UNDO deletion success",
+        "UNDO deletion failed")
+    deletion_redo = (
+        "REDO deletion success",
+        "REDO deletion failed")
+
+
+def AtomEditorComponents_HDRiSkybox_AddedToEntity():
+    """
+    Summary:
+    Tests the HDRi Skybox component can be added to an entity and has the expected functionality.
+
+    Test setup:
+    - Wait for Editor idle loop.
+    - Open the "Base" level.
+
+    Expected Behavior:
+    The component can be added, used in game mode, hidden/shown, deleted, and has accurate required components.
+    Creation and deletion undo/redo should also work.
+
+    Test Steps:
+    1) Create a HDRi Skybox with no components.
+    2) Add a HDRi Skybox component to HDRi Skybox.
+    3) UNDO the entity creation and component addition.
+    4) REDO the entity creation and component addition.
+    5) Enter/Exit game mode.
+    6) Test IsHidden.
+    7) Test IsVisible.
+    8) Delete HDRi Skybox.
+    9) UNDO deletion.
+    10) REDO deletion.
+    11) Look for errors.
+
+    :return: None
+    """
+
+    import azlmbr.legacy.general as general
+
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.utils import Report, Tracer, TestHelper
+    from Atom.atom_utils.atom_constants import AtomComponentProperties
+
+    with Tracer() as error_tracer:
+        # Test setup begins.
+        # Setup: Wait for Editor idle loop before executing Python hydra scripts then open "Base" level.
+        TestHelper.init_idle()
+        TestHelper.open_level("", "Base")
+
+        # Test steps begin.
+        # 1. Create a HDRi Skybox with no components.
+        hdri_skybox_entity = EditorEntity.create_editor_entity(
+            AtomComponentProperties.hdri_skybox())
+        Report.critical_result(Tests.hdri_skybox_entity_creation, 
+                               hdri_skybox_entity.exists())
+
+        # 2. Add a HDRi Skybox component to HDRi Skybox.
+        hdri_skybox_component = hdri_skybox_entity.add_component(
```

Comment:
yeah I noticed this on previous tests and mentioned it to Scott, he said it was ok that it was unused, and it could be used to easily extend the case in the future

Reply:
that's a fair point, we can leave it for now

Reply:
Also note that this same component variable exists in all P0 cases, so if I remove it here, we should remove it from every other P0 for consistency